diff --git a/node_modules/@react-native-community/progress-view/react-native-progress-view.podspec b/node_modules/@react-native-community/progress-view/react-native-progress-view.podspec
index 8be977e..55f0f5c 100644
--- a/node_modules/@react-native-community/progress-view/react-native-progress-view.podspec
+++ b/node_modules/@react-native-community/progress-view/react-native-progress-view.podspec
@@ -10,10 +10,11 @@ Pod::Spec.new do |s|
 
   s.authors          = package['author']
   s.homepage         = package['homepage']
-  s.platforms        = { :ios => "9.0", :osx => "10.14" }
+  s.platforms        = { :ios => "9.0", :osx => "10.14", :tvos => "9.0" }
 
   s.source           = { :git => "https://github.com/react-native-progress-view/progress-view.git", :tag => "#{s.version}" }
   s.ios.source_files = "ios/**/*.{h,m}"
+  s.tvos.source_files = "ios/**/*.{h,m}"
   s.osx.source_files = "macos/**/*.{h,m}"
 
   s.dependency 'React-Core'
