diff --git a/node_modules/react-native-file-access/react-native-file-access.podspec b/node_modules/react-native-file-access/react-native-file-access.podspec
index 6f28256..7a7292f 100644
--- a/node_modules/react-native-file-access/react-native-file-access.podspec
+++ b/node_modules/react-native-file-access/react-native-file-access.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "10.0", :osx => "10.10" }
+  s.platforms    = { :ios => "10.0", :osx => "10.10", :tvos => "10.0" }
   s.source       = { :git => "https://github.com/alpha0010/react-native-file-access.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm,swift}"
